adam_preprocessing/shape_stroke_extraction.py

```diff
@@ -996,16 +1085,16 @@ def main():
         total=curriculum_params["num_dirs"] if args.dir_num is None else 100,
     ):
         situation_dir = args.curriculum_path / f"situation_{situation_num}"
-        language_tuple: Tuple[str, ...] = tuple()
-        if (situation_dir / "description.yaml").exists():
+        language_tuple: Tuple[str, ...]
+        if (situation_dir / "description.yaml").exists() and not args.process_masks_independently:
             with open(
                 situation_dir / "description.yaml", encoding="utf-8"
             ) as situation_description_file:
                 situation_description = yaml.safe_load(situation_description_file)
             language_tuple = tuple(situation_description["language"].split(" "))
-            _integer_label, string_label = label_from_object_language_tuple(
-                language_tuple, situation_num
-            )
+            # _integer_label, string_label = label_from_object_language_tuple(
+            #     language_tuple, situation_num
+            # )
         else:
             string_label = "dummy"
```

Comment:
Might as well delete this dead code.

Reply:
To be clear, I think you can delete this whole `if`/`else` -- we don't actually need the string label so there's no point in loading it.